zerver/migrations/0542_rename_huddle_directmessagegroup_and_more.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 5.0.6 on 2024-07-05 10:44
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0541_alter_realmauditlog_options"),
+    ]
+
+    operations = [
+        migrations.RenameModel(
+            old_name="Huddle",
+            new_name="DirectMessageGroup",
+        ),
+        migrations.AlterModelTable(
+            name="directmessagegroup",
+            table="zerver_huddle",
+        ),
+    ]
```

Comment:
Does this migration generate any SQL via `manage.py sqlmigrate`? I worry that it might do something crazy, which could be avoided with advanced features like `state_operations`.

Reply:
Updated! `manage.py sqlmigrate`:
```
BEGIN;
--
-- Custom state/database change combination
--
-- (no-op)
COMMIT;
```

Reply:
Next time post what the original `sqlmigrate` output was too to just fully explain the trade-off and what changed; I was curious enough that I ended up checking myself.